corehq/apps/userreports/static/userreports/js/configure_report.js

```diff
@@ -0,0 +1,40 @@
+/* global reportBuilder */
+hqDefine('userreports/js/reportbuilder/configure_report', function() {
+    var initialPageData = hqImport('hqwebapp/js/initial_page_data');
+
+    window.ReportBuilder = {
+        Constants: hqImport("userreports/js/constants"),
+    };
+
+    $(function() {
+        var existing_report = initialPageData.get('existing_report'),
+            report_description = initialPageData.get('report_description');
+
+        var reportConfig = new reportBuilder.ReportConfig({
+            "columnOptions": initialPageData.get('column_options'),
+            "initialColumns": initialPageData.get('initial_columns'),
+            "app": initialPageData.get('application'),
+            "sourceId": initialPageData.get('source_id'),
+            "sourceType": "initialPageData.get('source_type')",
+            "reportPreviewUrl": "initialPageData.get('report_preview_url')",
+            "previewDatasourceId": "initialPageData.get('preview_datasource_id')",
+            "existingReport": existing_report ? existing_report.get_id : null,
```

Comment:
Does `existing_report.get_id` definitely work? `get_id` sounds like a function that wouldn't be available to the JSON-ified version of `existing_report`.

Reply:
Nope! Good call. It maps to ._id, which I've now changed it to